pyarlo/base_station.py

```diff
@@ -134,7 +134,7 @@ def __run_action(
         """
         url = NOTIFY_ENDPOINT.format(self.device_id)
 
-        body = ACTION_BODY
+        body = ACTION_BODY.copy()
```

Comment:
In my tests, this was inadvertently mutating the original `ACTION_BODY` dict between requests. Copying it to a local variable prevents that.

Reply:
Yes, that makes sense!! Good catch!